x-pack/plugins/security_solution/public/common/components/autocomplete_field/__examples__/index.stories.tsx

```diff
@@ -1,34 +0,0 @@
-/*
```

Comment:
This component isn't being used anywhere. Had spoken to @XavierM about deleting it.